Automattic/wp-calypso
# Remove the legacy PlansSingleProducts component

#### Changes proposed in this Pull Request

* Remove the `PlansSingleProducts` component since it's now replaced by the `ProductCard` component. 

#### Testing instructions

* Go to /plans page.
* Confirm no single product block based on the `PlansSingleProducts` is displayed there.

Fixes: n/a
